docs/docusaurus.config.ts

```diff
@@ -0,0 +1,161 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { VersionOptions } from "@docusaurus/plugin-content-docs";
+import * as Preset from "@docusaurus/preset-classic";
+import type { Config } from "@docusaurus/types";
+import { themes as prismThemes } from "prism-react-renderer";
+
+import DocsVersions from "./config/docs-versions.mjs";
+
+const includeLocalApiDocs = process.env.LOCAL_API_DOCS === "true";
+
+const githubUrl = "https://github.com/microsoft/FluidFramework";
+
+// TODO: set back to main before merging
+const githubMainBranchUrl = `${githubUrl}/tree/dev/docs/docusaurus`;
+const githubDocsUrl = `${githubMainBranchUrl}/docs`;
+
+// #region Generate the Docusaurus versions from our versions config.
+
+const versionsConfig: { [versionName: string]: VersionOptions } = {
+	current: {
+		label: DocsVersions.currentVersion.label,
+		badge: false,
+		banner: "none",
+	},
+};
+
+for (const version of DocsVersions.otherVersions) {
+	versionsConfig[version.version] = {
+		label: version.label,
+		path: version.path,
+		badge: true,
+		banner: "unmaintained",
```

Comment:
Is "unmaintained" the most accurate thing to say here? If I understand correctly the 1.x docs will be here, and while we don't intend to add new features there, I think we'd still be technically maintaining them?

Reply:
It's a good question. These notice configurations come out-of-the-box. The banner that this generates reads:

>This is documentation for Fluid Framework v1, which is no longer actively maintained.
> For up-to-date documentation, see the [latest version](https://lively-bay-0171da81e-preview.westus2.5.azurestaticapps.net/docs/) (v2).

This might be a bit aggressive in terms of messaging, but maybe not?

A couple of immediate options:
1. We can leave it be, assuming we're okay with our support messaging for v1 being somewhat aggressive.
2. We can remove these notices from the v1 docs

In either case, we can file follow-up work to create more tailored notices.

@rohandubal Curious what you think.